<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioRecorder</class>
 <widget class="QMainWindow" name="AudioRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="levelsLayout"/>
   </widget>
   <widget class="QLabel" name="levelLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>اکولایزر</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>60</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>مکث</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recordButton">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>60</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ضبط</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0">
     <item row="0" column="0">
      <widget class="QComboBox" name="audioDeviceBox"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>کارت صوتی:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="outputButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>فایل خروجی</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
